lcovrc

```diff
@@ -179,3 +183,7 @@ lcov_function_coverage = 1
 
 # Specify if branch coverage data should be collected and processed.
 lcov_branch_coverage = 0
+
+# Ask LCOV to return non-zero exit code is line coverage is below threshold
+# Default is 0.0 - i.e., do not check threshold.
```

Comment:
Typo: "...return non-zero exit code is line coverage..." => "... if line coverage..."